torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -466,63 +522,83 @@ def _export_fx_node_to_onnxscript(
                 )
             current_attr = getattr(current_attr, sub_attr_name)
 
-        input_ = onnxscript_graph.add_initializer(node.name, current_attr)
+        input_ = self.onnxscript_graph.add_initializer(node.name, current_attr)
 
-        assert isinstance(input_, graph_building.TorchScriptTensor)
+        assert isinstance(input_, onnxscript_graph_builder.TorchScriptTensor)
         assert isinstance(input_, onnxscript.tensor.Tensor)
-        fx_name_to_onnxscript_value[node.name] = input_
+        self.fx_name_to_onnxscript_value[node.name] = input_
         # FIXME: Refactor logic getting 'current_attr'.
         assert isinstance(current_attr, torch.Tensor)
-    else:
-        # TODO(wechi): Support get_attr, call_module, call_method.
-        raise RuntimeError(f"Found node type not defined in torch.fx: {node.op}")
 
 
-@_beartype.beartype
-@diagnostics.diagnose_call(diagnostics.rules.atenlib_fx_to_onnx)
-def export_fx_to_onnxscript(
-    diagnostic_context: diagnostics.DiagnosticContext,
-    fx_module_with_metadata: torch.fx.GraphModule,
-    options: ResolvedExportOptions,
-):
-    """
-    Export a torch.fx.GraphModule to a TorchScript graph with ONNX symbols.
+def _location_from_fx_stack_trace(
+    node_stack_trace: str,
+) -> Optional[diagnostics.infra.Location]:
+    """Extract location from FX node stack trace.
+
+    TODO(bowbao): Create fx utils module and move this function there.
 
     Args:
-        fx_module_with_metadata: A torch.fx.GraphModule with metadata.
-        export_options: Export options.
+        node_stack_trace: The stack trace of the FX node. Example:
+
+            File "path/file.py", line 311, in <function>
+                <code>
+            |   File "path/file2.py", line 389, in <function>
+                <code>
 
     Returns:
-        A TorchScript graph with ONNX symbols.
+        location: The location of the FX node.
     """
-    # Initialize the ONNX graph
-    onnxscript_graph = graph_building.TorchScriptGraph()
-    tracer = graph_building.TorchScriptTracingEvaluator(onnxscript_graph)
-
-    # In the following loop, a TorchScript graph is created to
```

Comment:
I don't see this comment is copied to somewhere in the new code. Please document the high-level idea in the new change. (probably not needed for this case if move all util functions to `staticmethod`).

Reply:
reviving this, I don't seem to find this comment in new code either.